products/esa-griml-iml-fv1/collection.json

```diff
@@ -0,0 +1,120 @@
+{
+    "type": "Collection",
+    "id": "esa-griml-iml-fv1",
+    "stac_version": "1.0.0",
+    "description": "This ice marginal lake dataset is a series of annual inventories, mapping the extent and presence of lakes across Greenland that share a margin with the Greenland Ice Sheet and/or the surrounding periphery glaciers. The annual inventories provide a comprehensive record of all identified ice marginal lakes, which have been detected using remote sensing techniques. The inventory series was created to better understand the impact of ice marginal lake change on the future sea level budget and the terrestrial and marine landscapes of Greenland, such as its ecosystems and human activities. The dataset is a complete inventory series of Greenland, with no absent data.The detected lakes are presented as polygon vector features in shapefile format, with coordinates provided in the WGS NSIDC Sea Ice Polar Stereographic North (EPSG:3413) projected coordinate system. Ice marginal lakes were identified using three independent remote sensing methods: 1) multi-temporal backscatter classification from Sentinel-1 synthetic aperture radar imagery; 2) multi-spectral indices classification from Sentinel-2 optical imagery; and 3) sink detection from the ArcticDEM (v3). All data were compiled and filtered in a semi-automated approach, using a modified version of the MEaSUREs GIMP ice mask (https://nsidc.org/data/NSIDC-0714/versions/1) to clip the dataset to within 1 km of the ice margin. Each detected lake was then verified manually. The methodology is open-source and provided at https://github.com/GEUS-Glaciology-and-Climate/GrIML for full reproducibility. Please consult the dataset readme provided with this dataset for further information on the associated metadata. Acknowledgements: The inventory series of ice marginal lakes in Greenland has been produced as part of the European Space Agency (ESA) Living Planet Fellowship project "Examining GReenland’s Ice Marginal Lakes under a changing climate (GrIML)", which is a follow-on effort to the 2017 inventory of ice marginal lakes created under the European Space Agency (ESA) Climate Change Initiative (CCI) in Option 6 of the Glaciers_cci project (4000109873/14/I-NB). Upkeep and continuation of the inventory series is supported by PROMICE, funded by the Geological Survey of Denmark and Greenland (GEUS) and the Danish Ministry of Climate, Energy and Utilities under the Danish Cooperation for Environment in the Arctic (DANCEA), conducted in collaboration with DTU Space (Technical University of Denmark) and Asiaq Greenland Survey. (2024-10-03)",

```

Comment:
The date in brackets at the end of this description does not need to be there. It is automatically generated by GEUS Dataverse. But I can understand keeping it if you would like this to follow exactly the description on the GEUS Dataverse.